### DISCUSSION TITLE
Kong Admin API and Kong Manager on KIC

### DISCUSSION THREAD

**Discussion by mikyll** [Kong/kubernetes-ingress-controller#5149] [Q&A] [answered]
How can I access and use the Kong Admin API/Kong Manager (OSS) on Kubernetes Ingress Controller? Should I use the Admin API on KIC, or there are any kind of drawbacks? Can someone provide some link to documentation or tutorials?

I have a cluster running locally with minikube:
```
NAME                                 TYPE           CLUSTER-IP       EXTERNAL-IP   PORT(S)                         AGE
kong-controller-validation-webhook   ClusterIP      10.102.174.162   <none>        443/TCP                         2d23h
kong-gateway-admin                   ClusterIP      None             <none>        8444/TCP                        2d23h
kong-gateway-manager                 NodePort       10.107.174.125   <none>        8002:30006/TCP,8445:32094/TCP   2d23h
kong-gateway-proxy                   LoadBalancer   10.107.202.167   127.0.0.1     80:30870/TCP,443:32591/TCP      2d23h
```

And I can access it through `minikube tunnel`

**Comment by programmer04** [comment, marked as answer]
Kubernetes YAMLs should be the single source of truth for Kong configuration on Kubernetes. Everything that is possible and makes sense should be doable with YAMLs. You don't need access to API or Kong Manager, so we don't expose it.

If you're looking for the UI, the supported solution is [Konnect](https://konghq.com/products/kong-konnect), you can read how to configure it with KIC [here](https://docs.konghq.com/konnect/gateway-manager/kic/#main).

> _If the above answers your question please click `Mark as answer` button to increase visibility for others._

**Reply by venkatb-zelar** [reply]
ok, but why is the manager service exposed in that case?
